galaxy/tools/deps/mulled/mulled_search.py

```diff
@@ -4,12 +4,21 @@
 import json
 import sys
 import tempfile
+import urllib2
+import logging
+from lxml import html
+from mulled_update_singularity_containers import get_singularity_containers
+
+#import subprocess
+import conda_api
```

Comment:
It would be nice if you could also comment on this @bgruening. For the conda API I used this: https://github.com/conda/conda-api. Perhaps there's a better alternative?

Reply:
I think this is great. We should use these libraries as much as we can.

Reply:
My concerns are 1) how to import it and 2) that it doesn't seem to be maintained.

Reply:
Oh now I see. Conda api is now part of the regular conda system.
For example see here: https://github.com/bioconda/bioconda-utils/blob/master/bioconda_utils/utils.py#L25

Requirements could be added to the testing system, like here: https://github.com/galaxyproject/galaxy-lib/blob/master/.travis.yml#L12

I will do it if you think this PR is ready, ok?
(given @jmchilton agrees and this PR is not totally out of scope)

Reply:
Okay, good to know.

I'll let you know when I'm ready.

Reply:
xref ContinuumIO/anaconda-verify#8 (comment) - I doubt they are going to allow Conda APIs to be used from PyPI installed projects so unofficial/unmaintained may be the best we get.

Reply:
Can we not use the conda installation from Galaxy? This project is really dead since a few years I think.